packages/connectivity/src/scp-cf/authorization-header.ts

```diff
@@ -156,6 +172,10 @@ interface AuthenticationHeaderOnPrem {
 interface AuthenticationHeaderProxy {
   'Proxy-Authorization': string;
 }
+interface AuthenticationNoAuthHeaders {
+  'Proxy-Authorization'?: string;
+  'SAP-Connectivity-Authentication'?: string;
+}
```

Comment:
[nth] Looking at the lines above and below, I have the impression that there is some potential for de-duplication.

Reply:
[req] I just noticed that some of the interfaces here are actually here twice! E.g. `AuthenticationHeaderOnPrem`. I know this was not part of the ticket, but please remove the duplicate types... 🤪